pandas/tests/indexing/test_floats.py

```diff
@@ -70,97 +80,107 @@ def test_scalar_error(self, index_func):
             tm.makePeriodIndex,
         ],
     )
-    def test_scalar_non_numeric(self, index_func):
+    @pytest.mark.parametrize("klass", [Series, DataFrame])
```

Comment:
Could be a logical follow up to make a `frame_or_series` fixture (we already have `index_or_series` in conftest) as used elsewhere for sure